kids-chatbot/src/components/ErrorBoundary.tsx

```
@@ -0,0 +1,50 @@
+            <div className="text-6xl mb-6">😔</div>
+            <h1 className="text-2xl font-bold text-red-600 mb-2">اوه! مشکلی پیش اومده</h1>
+            <p className="text-gray-600 mb-6">
+              متاسفم، یه خطای غیرمنتظره رخ داده. لطفاً صفحه رو تازه کن.
+            </p>
+            <button
+              onClick={() => window.location.reload()}
+              className="w-full bg-blue-500 hover:bg-blue-600 text-white font-medium py-3 px-4 rounded-lg transition-colors"
+            >
+              تازه کردن صفحه
+            </button>
+          </div>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)